```python
def plus(a,b):
  return a+b


plus(1,1,1,1,1,1,1,1,1) # 오류발생 인자 갯수 초과

```

```python
def plus(a,b,*args):
  print(args)
  return a+b

plus(1,2,1,1,1,1,1,1,1,1,1,1,1,1,1)
# (1,1,1,1,1,1,1) tuple형태로 출력됨

plus(1,2,1,1,1,1,1,1,1,1,1,1,1,1,1,hello=True, aa=True, df=True)
#오류

def plus(a,b, *args, **kwargs): # **kwargs추가
   print(args)
   print(kwargs)
	 return a+b

# argument 종류
# 1. position arguments
# 2. keyword arguments



def plus(*args):
  result = 0
  for number in args:
    result=number
	print(result)
  
plus(1,2,1,1,1,1,1,1,1,1,11,4,3,5,6,3)

# 출력
1
3
4
5
6
7
8
9
10
11
22
26
29
34
40
43
```



